### PR TITLE
Mention Detector for Docker Socket in dockershim removal FAQ

### PR DESCRIPTION
**Pull request by chris-short**
Adding the new Detector for Docker Socket plugin to the FAQ.

The tool will help folks find Dockershim usage in files on disk as well as running clusters.

It was developed and released through AWS' open source release process and we're recommending to customers use this tool.

### CODE REVIEW

File: content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 1]
```diff
@@ -200,6 +200,12 @@ discussion of the changes.
 
 [dep]: https://dev.to/inductor/wait-docker-is-deprecated-in-kubernetes-now-what-do-i-do-e4m
 
+### Is there any tooling that can help me find Dockershim in use
```
Comment (reylejano) [inline content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md:203]: nit: 
```suggestion
### Is there any tooling that can help me find dockershim in use
```

File: content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 2]
```diff
@@ -200,6 +200,12 @@ discussion of the changes.
...
+
+Yes! The [Detector for Docker Socket (DDS)][dds] is a kubectl plugin to detect if active Kubernetes workloads are mounting the docker socket (docker.sock) volume. Find more details and usage patterns in the [DDS project's README][dds]
```
Comment (nate-double-u) [inline content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md:205]: nit:
```suggestion
Yes! The [Detector for Docker Socket (DDS)][dds] is a kubectl plugin to detect if active Kubernetes workloads are mounting the docker socket (docker.sock) volume. Find more details and usage patterns in the [DDS project's README][dds].
```

File: content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md (hunk 1 above)
Comment (nate-double-u) [reply content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md:203]: I'm not sure why it would be different here. Why capitalize it?

**Comment by nate-double-u** [comment]
(oops, thought i could approve for blog stuff 😅)

File: content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 3]
```diff
@@ -200,6 +200,12 @@ discussion of the changes.
 
 [dep]: https://dev.to/inductor/wait-docker-is-deprecated-in-kubernetes-now-what-do-i-do-e4m
 
+### Is there any tooling that can help me find dockershim in use
+
+Yes! The [Detector for Docker Socket (DDS)][dds] is a kubectl plugin to detect if active Kubernetes workloads are mounting the docker socket (docker.sock) volume. Find more details and usage patterns in the [DDS project's README][dds].
```
Comment (sftim) [inline content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md:205]:
```suggestion
Yes! The [Detector for Docker Socket (DDS)][dds] is a kubectl plugin that you can
install and then use to check your cluster. DDS can detect if active Kubernetes workloads
are mounting the Docker Engine socket (`docker.sock`) as a volume.
Find more details and usage patterns in the DDS project's [README][dds].
```

**Review by sftim — COMMENTED** [review]
/hold

/approve

@chris-short you can either:
- /unhold
- tweak the wording and then get another LGTM (I'll nag folks if I need to)

**Comment by chris-short** [comment]
@sftim poke

**Comment by reylejano** [comment]
Reapplying lgtm
/lgtm
/hold cancel

**Pull request merged**